Recast/Source/Recast.cpp

```diff
@@ -71,6 +71,50 @@ void rcContext::log(const rcLogCategory category, const char* format, ...)
 	doLog(category, msg, len);
 }
 
+rcAreaModification::rcAreaModification(unsigned char value) :
+	m_value(value),
+	m_mask(RC_AREA_FLAGS_MASK)
```

Comment:
I mainly wanted 0xFF here (as a constant) since it makes sense that the mask has all bits set by default (even if the max value Detour supports is 6 bits set).